arangodb-net-standard/DocumentApi/Models/PatchDocumentsResponse.cs

```diff
@@ -1,11 +1,7 @@
 ﻿namespace ArangoDBNetStandard.DocumentApi.Models
 {
-    public class PatchDocumentsResponse<U> : DocumentBase
+    public class PatchDocumentsResponse<T> : PatchDocumentResponse<T>
```

Comment:
Are you sure the response should inherit from PatchDocumentResponse<T>?  I would expect for example mulitiple documents returned under `New` or `Old`, as opposed to a single document.

Reply:
The patch seems slightly different from the other endpoint. Here is an example with delete:

- method: `Task<DeleteDocumentsResponse<T>> DeleteDocumentsAsync<T>`
- `DeleteDocumentsResponse<T> : List<DeleteDocumentsDocumentResponse<T>>`
- `DeleteDocumentsDocumentResponse<T>: DeleteDocumentResponse<T>`

But for patch we have the following:

- method: `Task<IList<PatchDocumentsResponse<U>>> PatchDocumentsAsync<T, U>`
- `PatchDocumentsResponse<U> : DocumentBase`

I can take this opportunity to align it with other endpoints?

I find it more clear to return `IList<T>` explicitly, and have a model like: `PatchDocumentsSingleResponse`. But other endpoints are not like this.